python/cudf/cudf/_lib/parquet.pyx

```diff
@@ -675,15 +687,26 @@ cdef cudf_io_types.compression_type _get_comp_type(object compression):
         raise ValueError("Unsupported `compression` type")
 
 
-cdef _set_col_metadata(Column col, column_in_metadata& col_meta):
+cdef _set_col_metadata(
+    Column col,
+    column_in_metadata& col_meta,
+    object nullability
+):
+    if col.nullable and nullability is False:
```

Comment:
Had to check that `None` is not equal to `False. That's neat, thank you Python :)